.github/workflows/e2e.yml

```diff
@@ -0,0 +1,68 @@
+
+name: e2e
+
+# triggered by labeled PR
+on:
+  pull_request_target:
```

Comment:
This one is super dangerous in general, xref https://blog.gitguardian.com/github-actions-security-cheat-sheet/